core/src/components/datetime/test/a11y/datetime.spec.ts

```diff
@@ -0,0 +1,52 @@
+import type { SpecPage } from '@stencil/core/testing';
+import { newSpecPage } from '@stencil/core/testing';
+
+import { Item } from '../../../item/item';
+import { Datetime } from '../../datetime';
+
+describe('datetime', () => {
+  beforeEach(() => {
+    const mockIntersectionObserver = jest.fn();
+    mockIntersectionObserver.mockReturnValue({
+      observe: () => null,
+      unobserve: () => null,
+      disconnect: () => null,
+    });
+    global.IntersectionObserver = mockIntersectionObserver;
+  });
```

Comment:
Why is this needed?

Reply:
The `ion-datetime` implementation uses an`IntersectionObserver` but Jest does not have an implementation for it. You have to mock certain web APIs, such as `IntersectionObserver`, otherwise the spec test will throw an exception.